.github/workflows/pull-request.yml

```diff
@@ -46,14 +96,31 @@ jobs:
           CI: true
         run: make environment-setup
 
+      - name: Cache Mint packages
+        uses: actions/cache@v3
+        with:
+          path: .mint
+          key: ${{ runner.os }}-mint-${{ hashFiles('**/Mintfile') }}
+          restore-keys: |
+            ${{ runner.os }}-mint-
+
       - name: Build example
         run: make build-example
 
   test:
     runs-on: macos-latest
+    needs: install-mint
```

Comment:
Hi @kakcy , I think your PR looks good. One thing that makes me a little confused is why we need `install-mint` on the build, test, and build-example stages

Reply:
Hi @duyhungtnn , thank you for review!
In this PR, we added a Run Script to run "make run-lint" and run "swiftlint" when building the Bucketeer target.
(The purpose is to constantly check the analysis results during development.)
https://github.com/kakcy/bucketeer-ios-client-sdk/blob/change-how-swiftlint-manage/Bucketeer.xcodeproj/project.pbxproj#L1075

In "make run-lint", "swiftlint" is executed using "mint".
Therefore, since the Bucketeer target is built in each steps, I think "install-mint" is necessary.